src/singleton.js

```diff
@@ -72,5 +74,14 @@ export default class Singleton extends Tracer {
 
         // Carrier objects to be exposed at the package level
         this.BinaryCarrier = BinaryCarrier;
+
+        // XXX: what's the best way to export (and document) these two global
```

Comment:
It's a little odd since, for convenience, the package can be treated either as a singleton or a library (this is somewhat common in the JS world as I understand it).  Therefore, global functions actually should be added as proper methods on this `Singleton` class (i.e. same as `initNewTracer` and `initGlobalTracer`).